### DISCUSSION TITLE
Firebase Performance Monitoring currently doesn't support automatic network monitoring for Flutter apps.

### DISCUSSION THREAD

**Discussion by liuchuancong** [firebase/flutterfire#5566] [Feature request]
etc.

**Comment by ASemeniuk** [comment]
What is the status on this?
In my app I see that automatic nbetwork monitoring works perfectly in iOS
In Android, though, it tracks generic HTTP requests, but unfortunately not the request to FireStore and Cloud Functions. Is there a way to make it work on Android?
